### ISSUE TITLE
Youtube privacy consent information as media content

### ISSUE THREAD

**Issue by nilshoerrmann** [sylvainjule/kirby-embed#7]
When entering a video URL in the panel – instead of actual video data – the plugin stores Youtube's privacy consent information as media content:

```yml
----

Youtube:

input: >
  https://www.youtube.com/watch?v=ev-00si0KhM
media:
  title: Bevor Sie zu YouTube weitergehen
  description: null
  url: 'https://consent.youtube.com/ml?continue=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3Dev-00si0KhM&gl=DE&hl=de&pc=yt&uxe=23983172&src=1'
  type: link
  tags: [ ]
  image: null
  imageWidth: null
  imageHeight: null
  images: [ ]
  code: null
  feeds: [ ]
  width: null
  height: null
  aspectRatio: null
  authorName: null
  authorUrl: null
  providerIcon: https://www.google.com/favicon.ico
  providerIcons:
    - 
      url: https://www.google.com/favicon.ico
      width: 32
      height: 32
      size: 1024
      mime: image/x-icon
  providerName: youtube
  providerUrl: https://youtube.com
  publishedTime: null
  license: null

----
```

It seems to be this page that's being parsed which is currently shown when visiting Youtube for the first time from Germany (not sure about other parts of the world):

![grafik](https://user-images.githubusercontent.com/25466/113717874-59ebb580-96ec-11eb-8381-b667dad45464.png)

The panel displays a sync failed message:

<img width="443" alt="grafik" src="https://user-images.githubusercontent.com/25466/113717543-02e5e080-96ec-11eb-8cdb-f723583379ba.png">

**Comment by nilshoerrmann** [comment]
Related: https://github.com/oscarotero/Embed/issues/434

**Comment by benwest** [comment]
This is resolved in the core library, I'd be delighted to see that updated here - I'm too ignorant of Composer to do it myself 😐

**Comment by tristantbg** [comment]
Should be fixed in v1.0.3
https://github.com/sylvainjule/kirby-embed/commit/17b6640db6dac2920ffa93bd3a59aedbaf1bc5bd
